src/python/pants/core/goals/test.py

```diff
@@ -227,6 +229,19 @@ def register_options(cls, register) -> None:
                 "system supports this."
             ),
         )
+        register(
+            "--extra-env-vars",
+            type=list,
+            member_type=str,
+            default=[],
+            help=(
+                "Additional environment variables to include in test processes. "
+                "Entries are strings in the form `ENV_VAR=value` to use explicitly; or just "
+                "`ENV_VAR` to copy the value of a variable in Pants's own environment. `value` may "
+                "be a string with spaces in it such as `ENV_VAR=has some spaces`. `ENV_VAR=` sets "
+                "a variable to be the empty string."
+            ),
+        )
```

Comment:
Unsure whether it is worth including a big warning here for the caveat.

Reply:
I was planning on adding to the docs.